interface/main/tabs/js/user_data_view_model.js

```diff
@@ -92,6 +95,13 @@ function changePassword()
     });
 }
 
+function changeMFA()
+{
+    navigateTab(webroot_url+"/interface/usergroup/mfa_registrations.php","mfa", function () {
+        activateTabByName("mfa",true);
```

Comment:
set the target to 'mfa' in these 2 places:
https://github.com/openemr/openemr/blob/master/interface/main/tabs/menu/menus/standard.json#L1993
https://github.com/openemr/openemr/blob/master/sites/default/documents/custom_menus/Custom.json#L1993

Reply:
I thought the target attribute was to limit the number of menu items of that 'target' type that can be opened in a tab, if the target type changes then another tab opens with the new url. 

Perhaps I should change the function changeMFA() to 'msc'  instead

Reply:
agree that changing changeMFA() to 'msc' instead would make more sense